SDK/MobileCenterDistribute/Microsoft.Azure.Mobile.Distribute.iOS/Distribute.cs

```diff
@@ -8,7 +8,19 @@ namespace Microsoft.Azure.Mobile.Distribute
     public static partial class Distribute
     {
         [Preserve]
-        public static Type BindingType => typeof(iOSDistribute);
+        public static Type BindingType
+        {
+            get
+            {
+
+// We cannot use Conditional Attributes as they only work for non-void methods and classes, not for properties.
```

Comment:
You meant void instead of non-void?
In Crashes we have a conditional void method that works only in debug (generateTestCrashes).